client/src/worldgen.rs

```diff
@@ -210,8 +229,10 @@ struct EnviroFactors {
 impl EnviroFactors {
     fn varied_from(parent: Self, spice: u64) -> Self {
         Self {
-            max_elevation: parent.max_elevation + (1 - ((spice % 30) / 10) as i64),
-            temperature: parent.temperature + (1 - ((spice % 15) / 5) as i64),
+            max_elevation: parent.max_elevation
+                + (1 - ((spice % 30) / 10) as i64)
+                + (3 - parent.temperature.rem_euclid(7)),
```

Comment:
There should be a comment explaining the motivation behind making elevation dependent on temperature. Could the inverse be done instead? That seems more intuitive.

Reply:
temperature is just a stand-in for an abstract measure of slope.

Reply:
Then it should be renamed to suit its actual use. I think we still want an actual temperature measure to govern biomes, though, so maybe just add a new thing?